mobile-app/app/components/TransactionCardWalletTextInputV2.tsx

```diff
@@ -0,0 +1,211 @@
+import { forwardRef, useCallback } from 'react'
+import { Platform, StyleProp, Text, TextInputProps, TouchableOpacity, View, ViewStyle } from 'react-native'
+import { useBottomSheetInternal } from '@gorhom/bottom-sheet'
+import { ThemedIcon, ThemedProps, ThemedSectionTitleV2, ThemedTextInputV2, ThemedViewV2 } from '@components/themed'
+import { getColor, tailwind } from '@tailwind'
+
+type WalletTextInputProps = React.PropsWithChildren<TextInputProps> & IWalletTextInputProps
+export type InputType = 'default' | 'numeric'
+
+interface IWalletTextInputProps {
+  inputType: InputType
+  title?: string
+  titleTestID?: string
+  valid?: boolean
+  editable?: boolean
+  inlineText?: {
+    type: 'error' | 'helper'
+    text?: string | JSX.Element
+    style?: StyleProp<ViewStyle>
+  }
+  displayClearButton?: boolean
+  onClearButtonPress?: () => void
+  displayFocusStyle?: boolean
+  containerStyle?: string
+  inputContainerStyle?: StyleProp<ViewStyle>
+  onBlur?: () => void
+  hasBottomSheet?: boolean
+  inputFooter?: React.ReactElement
+  displayTickIcon?: boolean
+}
+
+export const TransactionCardWalletTextInputV2 = forwardRef<any, WalletTextInputProps>(function (props: WalletTextInputProps, ref: React.Ref<any>): JSX.Element {
```

Comment:
We added prefix `Wallet` to distinguish the `TextInput` component from usual text input, so let's follow that convention and rename component and file name to `WalletTransactionCardTextInput`?

I dont think we need a v2 since there is no v1